resources/assets/lib/components/user-card-brick.tsx

```diff
@@ -70,7 +78,17 @@ export default class UserCardBrick extends React.Component<Props> {
         <div className='user-card-brick__username u-ellipsis-overflow'>
           {this.props.user.username}
         </div>
+        {this.props.onRemoveClick != null && (
+          <div className='user-card-brick__remove'>
+            <span className='fas fa-times' onClick={this.handleRemoveClick}/>
+          </div>
```

Comment:
First this should be a button (otherwise can't be targeted with keyboard), and then this shouldn't be inside an `a` element.